### ISSUE TITLE
Replace raw resources

### ISSUE THREAD

**Issue by kmark** [rovo89/XposedBridge#53]
This would likely entail overriding `openRawResource` & co. in `XResources`. I might be able to submit this as a pull request at some point but I figured it's worth creating an issue beforehand.

**Comment by rovo89** [comment]
Yes, makes sense. I guess you're going to use a similar approach like `DrawableLoader`?
As I don't merge anything without thorough tests, it would also be good if you could set up a small example.

**Comment by kmark** [comment]
`DrawableLoader` is probably close to what needs to be done. The end goal is to allow either forwarding to another res ID or file path. Ideally there'd be a mechanism to load in raw bytes as well but that could get unnecessarily tricky when writing it out to a file and loading it back in would be the obvious userland workaround.

I'll be certain to supply a pre-built XposedBridge.jar and POC module in a separate repository when it's all said and done. From my understanding no modification to anything other than the Bridge would be required.
